docs/quickstarts/dotnet.md

```diff
@@ -3,6 +3,25 @@
 
 # .Net
 
-:::info
-.NET Quickstart guide coming soon
-:::
+It is simple to deploy a .NET Core application onto the FL0 platform. This example details getting started with ASP.NET Core - typically a standard configuration _should_ be able to be built with the [Paketo Buildpacks](https://github.com/paketo-buildpacks/) that are used.
+
+## Binding to the right port
```

Comment:
suggest to keep the wording here consistent with the other guides, eg https://github.com/fl0zone/fl0-docs/blob/main/docs/quickstarts/nodejs.md?plain=1#L8
```suggestion
## Listening to the Right Port
```